docs/submissions.md

```diff
@@ -291,20 +291,46 @@ Please be aware that titles created by fansubbers or scanlators are normally not
 
 ### Description
 
-A basic, spoiler-free description of the entry. This should be in English and a source provided \(whether taken from another site or written by yourself\).
+A basic, spoiler-free description of the entry. 
+
+It must be in English and a source should be provided if you took it from somewhere, so long as the source is one of the types accepted below.
+
+#### Acceptable sources
+
+Sources|Example
+---|---
+Official English licensor|Crunchyroll
+Reputable English news outlets|Anime News Network
+[Japanese publisher], translated|Kodansha, translated
+Collectives of people from reputable sites|MAL Rewrite
+
+**Note:** A reputable site is one unrelated to piracy or any other illegal activity.
+
+**Note 2:** [Japanese publisher] is a placeholder for the name of the publisher. Do not literally write "Japanese Publisher". This type of credit is also only for when the description is a direct translation from the publisher.
+
+#### Unacceptable sources 
```

Comment:
@Nogooduser – I can see this being misunderstood for "sources from whom we don't accept descriptions".
Perhaps these could be edited to "Credited Sources" and "Uncredited sources"?

Reply:
I can see the problem, I think creditable and non-creditable should solve the issue. Let me know if it still needs revision